website/static/download.html

```diff
@@ -42,47 +42,51 @@
 <div class="content homecontent width clearfix">
   <div>
     <h2>Current version</h2>
-    <p>The current Presto release version is <b id="presto_version_text">unset</b>.
-       <a id="presto_release_notes" href="broken">Release notes.</a>
+    <p id="presto_version_description">The current Presto release version is <b>0.220</b>.
+       See the <a id="presto_release_notes" href="broken">Release Notes</a> for more information. <br/>
+       This release features <b>120</b> commits from <b>22</b> contributors and <b>18</b> committers.
```

Comment:
I must be missing something. Are these hardcoded values displayed or replaced somewhere?

Reply:
Yes, there is javascript immediately following this that replaces them with the values from website/static/static/version.js

The old static site doesn't have a pre-rendering step and it doesn't have a server side rendering step so javascript it is. Once the page is converted to Docusarus the pre-rendering step can include version.js to get the correct values to render.